lerna.json

```diff
@@ -0,0 +1,6 @@
+{
+  "packages": [
+    "packages/*"
+  ],
+  "version": "1.0.1-alpha.7"
```

Comment:
@rodikh I think we should avoid releasing 1.x.x version.
Let's stick with minor for now.
The last one was 0.4.x. So we can use 0.5.x with "5" serving as non backward-compatible major.

Plus, I wish to avoid major "1" because we want to move to sourcebit anyway.

Reply:
also, set version to "independent"